js/pop-up.js

```diff
@@ -0,0 +1,147 @@
+    title: 'Multi-Post Stories',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-availability.png',
+    screenshotalt: 'Screenshot of Multi-Post Stories Project',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a challery of type and scrambled it to make a type specimen book. It has survived not only five centuries..',
+    links: ['html', 'css', 'javascript'],
+  },
+  {
+    id: 'display-weather-element',
+    title: 'Facebook 360',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-weather.png',
+    screenshotalt: 'Screenshot of Facebook 360 Project',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a challery of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    links: ['html', 'css', 'javascript'],
+  },
+  {
+    id: 'display-art-element',
+    title: 'Uber Navigation',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-art.png',
+    screenshotalt: 'Screenshot of Uber Navigation Project',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a challery of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    links: ['html', 'css', 'javascript'],
+  },
+  {
+    title: 'Window Pop-up',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-availability.png',
+    screenshotalt: 'Screenshot of Window pop-up Project',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a challery of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    links: ['html', 'css', 'javascript'],
+  },
+  {
+    title: 'Art Work',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-weather.png',
+    screenshotalt: 'Screenshot of Art Work Project',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a challery of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    links: ['html', 'css', 'javascript'],
+  },
+];
+
+const generateDetailsWork = (id, title, tags, screenshot, screenshotalt, description, links) =>
```

Comment:
Is this for the pop-up? To generate the pop-up?

Reply:
Yes sir, it's for the pop-up